### DISCUSSION TITLE
增加对Zotero7全文PDF翻译支持

### DISCUSSION THREAD

**Discussion by llity** [immersive-translate/immersive-translate#875] [Help]
### 概述 | Summary

Zotero7是基于firfox开发的文献管理工具，其中PDF有需要翻译的必要性，希望增加支持。
https://forums.zotero.org/discussion/105094/announcing-the-zotero-7-beta
https://www.zotero.org/support/beta_builds
https://www.zotero.org/support/dev/zotero_7_for_developers
https://github.com/windingwind/zotero-plugin-template
感谢！

**Comment by Objia** [comment]
附议，zotero太需要这个了

**Comment by liruifengv** [comment]
正在开发[BabelDOC](https://app.immersivetranslate.com/babel-doc/) 的 Zotero 插件
